apps/events/migrations/0013_create_event_categories.py

```diff
@@ -23,4 +23,4 @@ def create_event_categories(apps, schema_editor):
             ]
         )
 
-    operations = [migrations.RunPython(create_event_categories)]
+    # operations = [migrations.RunPython(create_event_categories)]
```

Comment:
Please remove this line if it will never be used again

Reply:
@Vadeux  could you be so kind as to add more details (sorry, if I am asking something silly, but really do not get why it will never be used again)?